tests/integration/components/registries/registration-list/component-test.ts

```diff
@@ -0,0 +1,62 @@
+import { click, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { module, test } from 'qunit';
+
+import Intl from 'ember-intl/services/intl';
+import { setupIntl } from 'ember-intl/test-support';
+import { setupRenderingTest } from 'ember-qunit';
+import { TestContext } from 'ember-test-helpers';
+import { OsfLinkRouterStub } from '../../../helpers/osf-link-router-stub';
+
+interface RegistrationListTestContext extends TestContext {
+    intl: Intl;
+}
```

Comment:
`import { TestContext } from 'ember-intl/test-support'`, adds `intl` to `ember-test-helpers`'s `TestContext`
```suggestion
```